portal/models/assessment_status.py

```diff
@@ -86,7 +86,7 @@ def qb_status_dict(user, questionnaire_bank):
     if not questionnaire_bank:
         return d
     trigger_date = questionnaire_bank.trigger_date(user)
-    start, _ = questionnaire_bank.calculated_start(trigger_date)
+    start, _ = questionnaire_bank.calculated_start(trigger_date, None)
```

Comment:
The default param for that second arg is None, no need to be explicit, right?

Reply:
Well, there's no default param for that arg in `calculated_start()`. Although you're right, probably easier to add it there.